life_server/Functions/MySQL/fn_bool.sqf

```diff
@@ -1,26 +1,20 @@
 /*
-    File: fn_bool.sqf
-    Author: TAW_Tonic
-
+    File:	fn_bool.sqf
+    Date:   2017-03-22 17:46:27
+    Author: Patrick "Lucian" Schmidt
+    
     Description:
-    Handles bool conversion for MySQL since MySQL doesn't support 'true' or 'false'
-    instead MySQL uses Tinyint for BOOLEAN (0 = false, 1 = true)
+    Converts Bool to Number and Number to bool
 */
-private ["_bool","_mode"];
-_bool = [_this,0,0,[false,0]] call BIS_fnc_param;
-_mode = [_this,1,0,[0]] call BIS_fnc_param;
 
-switch (_mode) do {
-    case 0: {
-        if (_bool isEqualType 0) exitWith {0};
-        if (_bool) then {1} else {0};
-    };
+params[
+    ["_in", "", [0,true]]
+];
+
+if (_in isEqualType true) exitWith {
+    [0,1] select _in;
+};
 
-    case 1: {
-        if (!(_bool isEqualType 0)) exitWith {false};
-        switch (_bool) do {
-            case 0: {false};
-            case 1: {true};
-        };
-    };
+if (_in isEqualType 0) exitWith {
+    [false,true] select _in;
```

Comment:
I took this code out from my MRES function which converts all Datatypes ready for DB thats why i used exitWith.
And tbh. what does it matter ?
This code is still 6x faster than the old shit. So use it or leave it. 

Here's the proof of the 6x faster performance, atleast on my system.
https://pastebin.com/qDe5mXZa